atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -46,6 +46,25 @@ def __init__(self, tasker, workflow, hub, target=None, koji_ssl_certs_dir=None):
             }
         self.xmlrpc = create_koji_session(hub, koji_auth_info)
 
+    def get_patched_release(self, original_release, increment=False):
+        if original_release.count('.') > 1:
+            release, suffix, remainder = original_release.split('.', 2)
+        else:
+            remainder = ''
+            if original_release.count('.') == 1:
+                release, suffix = original_release.split('.', 1)
+            else:
+                release = original_release
+                suffix = ''
+
+        result = [str(int(release) + 1)] if increment else [release]
```

Comment:
If `increment` is `True` then it should update as such.
Note that the first time this method is called, `increment` is `False.

It will only increment `20.f25` to `21.f25`, if `getNextRelease` has returned `20.f25` and for some reason that build already exists in Koji. In which case, this is an enhancement of a supposedly already handled scenario.